scripts/delayed.js

```diff
@@ -34,10 +34,14 @@ loadScript('/scripts/cookie-script.js');
 
 if (!isSidekickLibrary) {
   if (!defaultAnalyticsLoadDisabled()) {
-    // TODO: Remove delayed loading of GTM once it stops impacting page performance
-    setTimeout(() => {
-      loadGTM();
-    }, 2000);
+    if (getQueryParam('omitGTM') === '1') {
+      // Omit GTM
+    } else {
+      setTimeout(() => {
+        loadGTM();
+      }, 100);
```

Comment:
Apparently a delay as low as 100ms seems to suffice.

Reply:
@sdmcraft  may be we can have a have parameters something like 
`loadMartech = enum {ALL, Launch, GTM} `
We can have a better control what exactly will be loaded and keep this 100 ms delay as it is.  ?